app/src/main/AndroidManifest.xml

```diff
@@ -8,11 +8,23 @@
         android:dataExtractionRules="@xml/data_extraction_rules"
         android:fullBackupContent="@xml/backup_rules"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+        android:label="Easy Closet"
```

Comment:
in general it is good idea to have your strings as a resource in string.xml file. We do it mostly for localization purposes but it also helps organizing resources, see https://developer.android.com/guide/topics/resources/string-resource

Reply:
fixed